cocotb/log.py

```diff
@@ -135,6 +137,8 @@ class SimTimeContextFilter(logging.Filter):
     This uses the approach described in the :ref:`Python logging cookbook <python:filters-contextual>`.
 
     This adds the :attr:`~logging.LogRecord.created_sim_time` attribute.
+
+    .. versionadded:: 1.4
     """
 
     # needed to make our docs render well
```

Comment:
Could maybe make the docstring empty on the line below.

Reply:
This works indeed, will push later.